bgpd/bgp_fsm.c

```diff
@@ -289,7 +289,8 @@ void bgp_timer_set(struct peer *peer)
 		/* First entry point of peer's finite state machine.  In Idle
 		   status start timer is on unless peer is shutdown or peer is
 		   inactive.  All other timer must be turned off */
-		if (BGP_PEER_START_SUPPRESSED(peer) || !peer_active(peer)) {
+		if (BGP_PEER_START_SUPPRESSED(peer) || !peer_active(peer)
```

Comment:
We've had a long term standing internal issue with attempting to start connections before a vrf is brought to a state that is usable.  I bet this fixes the issue for us too.  Awesome

Reply:
yes. I hope it too.